src/example/App/VariableText.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import createReactClass from 'create-react-class';
 import {shouldComponentUpdate} from 'react/lib/ReactComponentWithPureRenderMixin';
```

Comment:
Can only be used with ES6 Classes, not `createReactClass` factory =(

Need to update all the code to use Classes instead, to avoid extra dependency.